src/coinbase/types.ts

```diff
@@ -244,6 +249,77 @@ export type WalletAPIClient = {
   ): AxiosPromise<WalletList>;
 };
 
+/**
+ * SmartWalletAPI client type definition.
+ */
+export type SmartWalletAPIClient = {
```

Comment:
`SmartWalletAPIClient` feels a little long to me, maybe just `SmartWalletClient` or `SmartWallet` if this is the entrypoint for devs to broadcast transactions might make sense